docs/pbc/all/tax-management/202311.0/base-shop/third-party-integrations/vertex/install-vertex.md

```diff
@@ -235,7 +281,35 @@ class MessageBrokerDependencyProvider extends SprykerMessageBrokerDependencyProv
 
 ```
 
-### 5. Optional: To send invoices to Vertex through OMS, configure your Payment OMS
+### 5. Configure the OMS dependency provider
+
+Add the following code to `src/Pyz/Zed/Oms/OmsDependencyProvider.php`:
+
+```php
+
+namespace Pyz\Zed\Oms;
+
+use Spryker\Zed\Oms\OmsDependencyProvider as SprykerOmsDependencyProvider;
+use Spryker\Zed\TaxApp\Communication\Plugin\Oms\OrderRefundedEventListenerPlugin;
+
+class OmsDependencyProvider extends SprykerOmsDependencyProvider
+{
+// ...
+
+    /**
+     * @return array<\Spryker\Zed\OmsExtension\Dependency\Plugin\OmsEventTriggeredListenerPlugin>
+     */
+    protected function getOmsEventTriggeredListenerPlugins(): array
+    {
+        return [
+            new OrderRefundedEventListenerPlugin(),
```

Comment:
Please add a note, that this plugin expects a specific state name in the OMS configuration to work.

Maybe can be merged with #6 to the section `Configure your OMS process.`

Reply:
Section 6 is marked as optional as it covers invoices. I assume that we aren't marketing refunds as optional feature?

Reply:
Please check it with Oluwanifemi Omotola

Reply:
Refunds are optional, moved the config section